backend/src/schema/resolvers/posts.js

```diff
@@ -79,6 +79,7 @@ export default {
       delete params.categoryIds
       params = await fileUpload(params, { file: 'imageUpload', url: 'image' })
       params.id = params.id || uuid()
+      params.createdAt = params.createdAt || new Date().toISOString()
```

Comment:
We should really always working with adding utc dates to avoid issues down the road. 

Also: why the heck is it possible to overwrite the createdAt date, this should never, ever happen.

Reply:
good question... I pushed up some code that would set the `created_at` in a cypher statement, and then parse the `post.created_at` to return a String as `createdAt`... it's not done yet, but I am for the day... 
would you and/or @roschaefer have a look and tell me if you don't like it/want to go in a different direction?